src/material-examples/table-expandable-rows/table-expandable-rows-example.css

```diff
@@ -7,7 +7,7 @@ tr.example-detail-row {
 }
 
 tr.example-element-row:not(.example-expanded-row):hover {
-  background: #f5f5f5;
+  background: #cccccc;
```

Comment:
I played around with a contrast ratio calculator and it looks like `#777` will actually be the best compromise between light and dark

Reply:
Thank you for looking at this. I've changed the color to #777.